### PR TITLE
updated user auth_method to camelCase

### PR DESCRIPTION
**Pull request by saberlynx**
updated variables to camelCase. Everything seems to work from manual testing. I ignored the commands/users code because changing it breaks the functionality to change password. Also ignored parts of dialog.
**Checklist**:

- [] Tests
- [] [CHANGELOG](https://github.com/greenbone/gsa/blob/master/CHANGELOG.md) Entry

### CODE REVIEW

File: gsa/src/web/pages/users/row.js [hunk 1]
```diff
@@ -109,7 +109,8 @@ const Row = ({
     </DetailsLink>
   ));
 
-  const auth_method = convert_auth_method(entity.auth_method);
+  const authMethod = convert_auth_method(entity.authMethod);
+  delete entity.authMethod;
```
Comment (bjoernricks) [inline gsa/src/web/pages/users/row.js]:
```suggestion
```

File: gsa/src/gmp/models/user.js [hunk 2]
```diff
@@ -77,13 +77,13 @@ class User extends Model {
...
+        ret.authMethod = AUTH_METHOD_LDAP;
       } else if (source === 'radius_connect') {
-        ret.auth_method = AUTH_METHOD_RADIUS;
+        ret.authMethod = AUTH_METHOD_RADIUS;
       }
       delete ret.sources;
     } else {
-      ret.auth_method = AUTH_METHOD_PASSWORD;
+      ret.authMethod = AUTH_METHOD_PASSWORD;
     }
```
Comment (bjoernricks) [inline gsa/src/gmp/models/user.js]:
```suggestion
    delete ret.auth_method;
```

File: gsa/src/gmp/models/__tests__/user.js [hunk 3]
```diff
@@ -118,10 +118,10 @@ describe('User model tests', () => {
     const user2 = new User(elem2);
     const user3 = new User({});
 
-    expect(user1.auth_method).toEqual(AUTH_METHOD_LDAP);
-    expect(user2.auth_method).toEqual(AUTH_METHOD_RADIUS);
+    expect(user1.authMethod).toEqual(AUTH_METHOD_LDAP);
+    expect(user2.authMethod).toEqual(AUTH_METHOD_RADIUS);
     expect(user1.sources).toBeUndefined();
-    expect(user3.auth_method).toEqual(AUTH_METHOD_PASSWORD);
+    expect(user3.authMethod).toEqual(AUTH_METHOD_PASSWORD);
```
Comment (bjoernricks) [inline gsa/src/gmp/models/__tests__/user.js]: We could even add a check for having converted the auth_method property to the tests :-)

```suggestion
    expect(user3.auth_method).toBeUndefined()
```

Comment (saberlynx) [reply gsa/src/gmp/models/__tests__/user.js]: Looking at model data from the tests, there doesn't appear to be an original auth_method. we determined the authMethod based on a property called "sources". Perhaps something like this? `expect(user3.sources).toBeUndefined();`

Comment (bjoernricks) [reply gsa/src/gmp/models/__tests__/user.js]: Ah ok. It isn't parsed from an auth_method property. So we don't need another test for this :-)

**Pull request merged**